packages/earth-map/src/utils/store.ts

```diff
@@ -0,0 +1,32 @@
+import { Store } from 'redux';
+
+import { setPlacesSearch } from 'modules/places/actions';
+import { setUserGroup } from 'modules/user/actions';
+import { setMapStyle } from 'modules/map/actions';
+import { setLastViewedPlace } from 'modules/global/actions';
+
+/**
```

Comment:
why is this here? there's a folder store, utils should have small helper functions